pytest_plugins/manual_skipped.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+import pytest
+
+from robottelo.constants import NOT_IMPLEMENTED
+
+LOGGER = logging.getLogger(__name__)
+
+
+def pytest_configure(config):
+    """Register custom marker for stubbed test cases."""
+    marker = "stubbed: Tests that are not automated yet or manual only."
+    config.addinivalue_line("markers", marker)
+
+
+def pytest_collection_modifyitems(session, config, items):
+    """
+    Collects tests based on pytest option to select tests marked as failed on Report Portal
+    """
+    manualskip_args = config.getvalue('mark_manuals_skipped')
+    if not manualskip_args:
+        return
+    for item in items:
+        if item.get_closest_marker(name='stubbed'):
+            item.add_marker(marker=pytest.mark.skip(reason=NOT_IMPLEMENTED))
+
+
+def pytest_addoption(parser):
+    """Add options to pytest to modify manual tests as skipped"""
+    help_script = 'Mark stubbed tests as skipped tests.'
+    parser.addoption("--mark-manuals-skipped", action='store_true', default=True, help=help_script)
```

Comment:
Discussed with @jyejare updating this option to allow for controlling the stubbed case results between passing and skipping.  Default should be skipping, as that's the behavior that we want when these cases are included in collection by default.  The option here should reflect that by having a default boolean that will skip when there is no option passed. Passing the option should trigger the non-default behavior of `passed` results for stubbed cases.

Reply:
@mshriver Done! 

All the files are also replaced to use `stubbed` marker from pytest directly instead of from `robottelo.decorators`